app/components/trn_details_component.rb

```diff
@@ -91,4 +91,10 @@ def previous_name
       @trn_request.previous_name
     end
   end
+
+  def trn_from_user_value
+    return @trn_request.trn_from_user if session[:user_knows_trn] == "true"
+
+    "I don’t know my TRN"
```

Comment:
Could you provide a screenshot in the PR description with what this looks like so that our designers can give it a look?

Reply:
This has now been provided in the PR description